### PR TITLE
Few fixed related to marko support

### PR DESCRIPTION
**Pull request by nm123github**
Issue:

## What I did
- Fix some marko cli issues.
- Also, add some new examples in default.

## How to test

Is this testable with Jest or Chromatic screenshots?
Not sure

Does this need a new example in the kitchen sink apps?
Added new example.

Does this need an update to the documentation?
No

### CODE REVIEW

File: lib/cli/generators/MARKO/template/.storybook/addons.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
 import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
```
Comment (Hypnosphi) [inline lib/cli/generators/MARKO/template/.storybook/addons.js]: You need to add it as dependency in `MARKO/index.js` then

**Review by ndelangen — APPROVED** [review]
Great work

File: lib/cli/generators/MARKO/template/.storybook/addons.js (hunk 1 above)
Comment (nm123github) [reply lib/cli/generators/MARKO/template/.storybook/addons.js]: Done 👍

Comment (Hypnosphi) [reply lib/cli/generators/MARKO/template/.storybook/addons.js]: Please also commit the new files in `snapshots/marko/components` (`git add -A` should help)

Comment (nm123github) [reply lib/cli/generators/MARKO/template/.storybook/addons.js]: Yes, snapshots files are part of PR.

**Review by Hypnosphi — APPROVED** [review]

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.8`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.8)

**Pull request merged**